### ISSUE TITLE
Write tables to HDFS

### ISSUE THREAD

**Issue by bmyerz** [uwescience/myria#729]
Motivation: I want a way to get large tables out of Myria.

Depends on moving serialization out to the workers: #705

**Comment by domoritz** [comment]
Also parquet #482 would be nice for fast read/write in columnar format.

**Comment by bmyerz** [comment]
Yes, Parquet is a more appropriate storage backend since it remains table-like. The main use case that I want to support with HDFS is bulk transfer or sharing of large datasets. If Parquet satisfies that and has an easy client to interface with then it fits the bill.

**Comment by senderista** [comment]
See also Postgres support for ORCFile: https://github.com/gokhankici/orc_fdw
